File: Android/12 - SuperMario/app/src/main/java/com/example/supermario/Cadastro.kt
package com.example.supermario

class Cadastro {
    private lateinit var lista: ArrayList<Pessoa>

    init {
        this.lista = arrayListOf()
    }

    fun add(pessoa: Pessoa){
        this.lista.add(pessoa)
    }

    fun get(index: Int) : Pessoa{
        return this.lista.get(index)
    }

    fun get(): ArrayList<Pessoa>{
        return this.lista
    }

    fun count(): Int{
        return this.lista.count()
    }
}

File: iOS/06 - Amigo Secreto/Amigo Secreto/model/Cadastro.swift
//
//  Cadastro.swift
//  Amigo Secreto
//
//  Created by Valéria Cavalcanti on 05/12/19.
//  Copyright © 2019 Valéria Cavalcanti. All rights reserved.
//

import Foundation

class Cadastro: NSObject {
    var lista: Array<Confra>
    
    override init() {
        self.lista = Array<Confra>()
    }
    
    func add(confra: Confra) {
        self.lista.append(confra)
    }
    
    func get() -> Array<Confra> {
        return self.lista
    }
    
    func get(index: Int) -> Confra {
        return self.lista[index]
    }
    
    func del(index: Int) {
        self.lista.remove(at: index)
    }
    
    func count() -> Int {
        return self.lista.count
    }
}
